### DISCUSSION TITLE
Security Onion 2.3.240 now available!

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#10235] [Announcements]
https://blog.securityonion.net/2023/04/security-onion-23240-now-available.html

![40_dashboards](https://user-images.githubusercontent.com/1659467/234673830-0ca56c07-72b1-495b-bff2-ae7804fac325.png)

**Comment by smallboy69** [comment]
Awesome stuff Security Onion 2 ...can please advise where i can download just the security Onion without centos 7 as i have installed Ubuntu 20.4 and just need to install Security Onion 2 application...

**Reply by dougburks** [reply]
If you want to install on Ubuntu, please follow the instructions in the documentation:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

If you have further questions or problems, please start a new discussion rather than replying to this announcement.
